src/screens/Booking/components/PassengerForm/Form.tsx

```diff
@@ -0,0 +1,101 @@
+import {useFormik} from 'formik';
+import React from 'react';
+import * as Yup from 'yup';
+import {t} from '@translations';
+import {useDispatch} from 'react-redux';
+import {AppDispatch} from '@store/store';
+import {buy} from '@modules/ticket';
+import {View, Text, TouchableOpacity} from 'react-native';
+import {InputField} from '@components';
+import styles from './Form.styles';
+
+interface FormProps {
+  flightId: string;
+}
+
+const Form = ({flightId}: FormProps) => {
+  const initialValues = {
+    name: '',
+    surname: '',
+    passport: '',
+  };
+
+  const validationSchema = Yup.object().shape({
```

Comment:
For the name and surname validations, I suggest using Yup's string.matches schema and passing in a regular expression that allows only letters. Otherwise, numbers are treated as if they are valid here.